cistem/protocols/protocol_tomo_resample.py

```diff
@@ -0,0 +1,184 @@
+# **************************************************************************
+# *
+# * Authors:     Ricardo D. Righetto (ricardo.righetto@unibas.ch)
+# *
+# * University of Basel
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+
+'''
+A protocol to resample tomograms by Fourier cropping/padding using cisTEM.
+'''
+from cistem import Plugin
+from pwem.protocols import EMProtocol
+from pyworkflow import BETA
+from pyworkflow.protocol import PointerParam, IntParam
+from pyworkflow.utils import *
+from tomo.objects import SetOfTomograms, Tomogram
+from pwem.convert.headers import Ccp4Header
+
+OUTPUT_TOMO_NAME = 'resampledTomos'
+OUTPUT_DIR = 'extra/'
+
+class ProtTomoResample(EMProtocol):
```

Comment:
You would probably want to inherit not from EMProtocol (too general), but from ProtTomoBase instead. This will also make outputs creation easier, e.g. simply self._createSetOfTomograms. See tomo/protocols/protocol_base.py